langlib/lang.table/src/main/ballerina/table.bal

```diff
@@ -86,6 +115,29 @@ public isolated function get(table<MapType> key<KeyType> t, KeyType k) returns M
 # otherwise, it will be added as the last member.
 # It panics if parameter `val` is inconsistent with the inherent type of parameter `t`.
 #
+# ```ballerina
+# table<record {|readonly int id; string name;|}> key(id) students = table [
+#         {id: 1, name: "Jo"},
+#         {id: 2, name: "Sam"}
+#     ];
+# 
+# students.put({id: 3, name: "Pat"});
+# students ⇒ [{"id":1,"name":"Jo"},{"id":2,"name":"Sam"},{"id":3,"name":"Pat"}]
+# 
+# students.put({id: 1, name: "Kane"});
+# students ⇒ [{"id":1,"name":"Kane"},{"id":2,"name":"Sam"},{"id":3,"name":"Pat"}]
+# 
+# var tbl = table [
```

Comment:
Why are we using `var`?

Reply:
This is a suggestion I made. Since it's a keyless table, it's fine to use `var` right?

Reply:
In here we are creating a keyless table. The easiest way of doing that is this way.

Reply:
Not significantly simple than something like

```ballerina
table<record {|int id; string name;|}> tbl = table [
    {id: 1, name: "Jo"},
    {id: 2, name: "Sam"}
];
```

though, right?

Because we already have the type for the key-ed table.

Reply:
Yes true, but again if the example doesn't need to emphasize the tables' constraint; I think we don't actually need it even. Since it's an example using `var` should be fine I guess. But then again, do we have a verdict on not using `var` in the examples?

Reply:
IMO, it would be clearer to use the type, since the user wouldn't have to figure it out by looking at the constructor.

Reply:
+1